fastdds_python/test/api/test_domainparticipant.py

```diff
@@ -22,6 +22,28 @@ def __init__(self):
         super().__init__()
 
 
+def test_contains_entity():
+    """
+    This test checks:
+    - DomainParticipant::contains_entity
+    """
+    factory = fastdds.DomainParticipantFactory.get_instance()
+    assert(factory is not None)
+    participant = factory.create_participant(
+            0, fastdds.PARTICIPANT_QOS_DEFAULT)
+    assert(participant is not None)
+
+    publisher = participant.create_publisher(fastdds.PUBLISHER_QOS_DEFAULT)
+    assert(publisher is not None)
+
+    assert(participant.contains_entity(publisher.get_instance_handle()))
```

Comment:
Could we have a negative test here? For instance, checking that `contains_entity` returns False for a default constructed instance handle.